loadtest/loadtest.go

```diff
@@ -19,6 +19,7 @@ type LoadTester struct {
 	config        *Config
 	wg            sync.WaitGroup
 	statusChan    chan control.UserStatus
+	quitChan      chan struct{}
```

Comment:
Interesting. My view of this race was that we just needed to make a safe copy of `statusChan` in `handleStatus()` by leveraging the existing `RWMutex` and range on that copy.  
The goroutine should already exit as a result of closing `statusChan` so adding an additional channel to quit seems counter-intuitive. What am I missing?

Reply:
Hmm .. I seriously need to stop over-thinking. Yeah, that works obviously.